src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/OperatorSymbolExtensions.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    [Flags]
+    internal enum OperatorPosition
+    {
+        None = 0,
+        Prefix = 1,
+        Infix = 2,
+        Postfix = 4,
+    }
+
+    internal static class OperatorSymbolExtensions
+    {
+        internal static string GetOperatorSignOfOperator(this IMethodSymbol m)
+        {
+            return m.Name switch
+            {
+                // binary
+                WellKnownMemberNames.AdditionOperatorName => "+",
+                WellKnownMemberNames.BitwiseAndOperatorName => "&",
+                WellKnownMemberNames.BitwiseOrOperatorName => "|",
+                WellKnownMemberNames.DivisionOperatorName => "/",
+                WellKnownMemberNames.EqualityOperatorName => "==",
+                WellKnownMemberNames.ExclusiveOrOperatorName => "^",
+                WellKnownMemberNames.GreaterThanOperatorName => ">",
+                WellKnownMemberNames.GreaterThanOrEqualOperatorName => ">=",
+                WellKnownMemberNames.InequalityOperatorName => "!=",
+                WellKnownMemberNames.LeftShiftOperatorName => "<<",
+                WellKnownMemberNames.LessThanOperatorName => "<",
+                WellKnownMemberNames.LessThanOrEqualOperatorName => "<=",
+                WellKnownMemberNames.ModulusOperatorName => "%",
+                WellKnownMemberNames.MultiplyOperatorName => "*",
+                WellKnownMemberNames.RightShiftOperatorName => ">>",
+                WellKnownMemberNames.SubtractionOperatorName => "-",
+
+                // Unary
+                WellKnownMemberNames.DecrementOperatorName => "--",
+                WellKnownMemberNames.FalseOperatorName => "false",
+                WellKnownMemberNames.IncrementOperatorName => "++",
+                WellKnownMemberNames.LogicalNotOperatorName => "!",
+                WellKnownMemberNames.OnesComplementOperatorName => "~",
+                WellKnownMemberNames.TrueOperatorName => "true",
+                WellKnownMemberNames.UnaryNegationOperatorName => "-",
+                WellKnownMemberNames.UnaryPlusOperatorName => "+",
+
+                _ => throw ExceptionUtilities.UnexpectedValue(m.Name),
+            };
+        }
+
+        public static string GetOperatorName(this IMethodSymbol m)
+        {
+            return m.Name switch
+            {
+                // binary
+                WellKnownMemberNames.AdditionOperatorName => "a + b",
```

Comment:
when and how does this appear?

Reply:
It does not appear any longer (I had it as part of "DisplayText", but this was no good). I think you should define:
* which operators should be suggested (I already excluded "true" and "false")
* which types should be excluded (e.g. decimal offers 12 operators and a couple of conversions, see screencast, I think these pollute the completion list)
* in which order should the operator appear (currently alphabetical per sign, see screencast)
* how  should the description be presented (currently `=> SymbolCompletionItem.GetDescriptionAsync(item, document, cancellationToken);` see screencast in the PR description). A proposal for descriptions is also needed for indexers and conversions. Conversions for instance could be described as `"Explicit conversion to" + GetDescriptionAsync(typeSymbolOfConversionTarget)`.

![FipAJm9c4f](https://user-images.githubusercontent.com/24472128/94473910-2411c580-01cd-11eb-8826-3b14276d293d.gif)

Reply:
> which types should be excluded (e.g. decimal offers 12 operators and a couple of conversions, see screencast, I think these pollute the completion list)

that actually looks totally fine to me... :)